packages/cli/src/ceramic-daemon.ts

```diff
@@ -143,22 +143,21 @@ class CeramicDaemon {
   }
 
   registerAPIPaths (app: core.Express, gateway: boolean): void {
-    app.get(toApiPath('/records/ceramic/:docid'), this.records.bind(this))
-    app.get(toApiPath('/show/ceramic/:docid'), this.show.bind(this))
-    app.get(toApiPath('/state/ceramic/:docid'), this.state.bind(this))
-    app.get(toApiPath('/pin/ls/ceramic/:docid'), this.listPinned.bind(this))
-    app.get(toApiPath('/pin/ls'), this.listPinned.bind(this))
-    app.get(toApiPath('/supported_chains'), this.getSupportedChains.bind(this))
-    app.post(toApiPath('/create'), this.createDocFromGenesis.bind(this))
+    app.get(toApiPath('/records/:docid'), this.records.bind(this))
+    app.put(toApiPath('/document'), this.createDocFromGenesis.bind(this))
+    app.get(toApiPath('/document/:docid'), this.state.bind(this))
+    app.get(toApiPath('/pins/:docid'), this.listPinned.bind(this))
+    app.get(toApiPath('/pins'), this.listPinned.bind(this))
+    app.get(toApiPath('/node/chains'), this.getSupportedChains.bind(this))
```

Comment:
Should we refactor `/node/chains` into `/node` or `/node/config`? In that way we don't need to change the API if we want to add more info instead of supported chains.